### DISCUSSION TITLE
how to change colors?

### DISCUSSION THREAD

**Discussion by mahesh11T** [zauberzeug/nicegui#1369] [Q&A] [answered]
### Question

```py
from nicegui.elements import colors
colors.Colors(primary='#0a3090')
```

**Comment by falkoschindler** [comment, marked as answer]
Please be more specific. What would you like to do? Have you read https://nicegui.io/documentation/colors?
We really try to respond to every discussion/issue and it would help if you take some time to formulate your idea.

**Reply by mahesh11T** [reply]
sorry, I did not take look at it. thanks
